src/App.tsx

```diff
@@ -172,6 +172,15 @@ function AppContent() {
   const [plotNames, setPlotNames] = useState<Record<number, string>>({});
   const [showNavBar, setShowNavBar] = useState(true);
 
+  const setDefaults = useCallback(() => {
+    if (!isPlaceholderOrg(selectedOrganization.id)) {
+      const savedColumns = orgPreferences.accessionsColumns ? (orgPreferences.accessionsColumns as string[]) : [];
+      const defaultColumns = savedColumns.length ? savedColumns : DefaultColumns(preferredWeightSystem).fields;
```

Comment:
this fixes the case where user preferences are not restored  on reloading the accessions table